### ISSUE TITLE
metadata: patch for Relationship Dialogue Overhaul + AI Overhaul incorrect filename

### ISSUE THREAD

**Issue by TheGameBall** [loot/skyrimse#2228]
The actual patch filename does not contain a dash separator (and of course, only one space between the two words). This causes the "you haven't enabled" message to show.

This is incorrect: 
        {
            "condition": "active(\"AI Overhaul.esp\") and not active(\"AI Overhaul - Relationship Dialogue Overhaul Patch.esp\")",
            "language": "en",
            "text": "You seem to be using **[AI Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/21654/)**, but you have not enabled a compatibility patch for this mod. A third party patch is available here: [AI Overhaul - Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/35823/)",
            "type": "say"
        },

This is correct (unless we want to ask the patch maker to rename his file):
        {
            "condition": "active(\"AI Overhaul.esp\") and not active(\"AI Overhaul Relationship Dialogue Overhaul Patch.esp\")",
            "language": "en",
            "text": "You seem to be using **[AI Overhaul](https://www.nexusmods.com/skyrimspecialedition/mods/21654/)**, but you have not enabled a compatibility patch for this mod. A third party patch is available here: [AI Overhaul - Official Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/35823/)",
            "type": "say"
        },

**Comment by sibir-ine** [comment]
The RDO patch from _AI Overhaul SSE - Official Patch Hub_ is named _AI Overhaul - Relationship Dialogue Overhaul Patch.esp_, which is indeed what the condition for that message currently looks for. Is there another patch available from another page named _AI Overhaul Relationship Dialogue Overhaul Patch.esp_? Or is it an older, no longer available version of the patch or a personal patch? If it's the former, & if you link to it, we can add it in the condition too. If it's the latter, you'll have to either switch to the patch from the aforementioned page or just ignore the message (it's not a warning or an error at least).

**Comment by TheGameBall** [comment]
Hmmm... I double-checked the RAR file, and it has the dash. I can only assume that somehow I went in there and renamed it myself for no good reason, forgot about it, and then discovered I had to rename it (again, it turns out), or used some weird alternative extraction process (I don't have WinRAR installed). I went and redownloaded, and reinstalled the file, and went looking for other versions online, and couldn't find any version missing the dash and space, or any alternate/older versions at all, so it must have been one heck of a misclick.

I don't know how it came to be that I could have done this, but I must have, there is no other explanation. Sorry for the waste of time.
